src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/Razor/RazorProvideDynamicFileResponse.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json.Serialization;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.HostWorkspace.Razor;
+
+internal class RazorProvideDynamicFileResponse
+{
+    [JsonPropertyName("csharpDocument")]
+    public required TextDocumentIdentifier CSharpDocument { get; set; }
+
+    [JsonPropertyName("edits")]
+    public ServerTextChange[]? Edits { get; set; }
+
+    [JsonPropertyName("checksum")]
+    public required byte[] Checksum { get; set; }
```

Comment:
Did this need to change to `string`? or did you work that out?

Reply:
oh woops. Have it local and didn't push